container/container_test.go

```diff
@@ -116,19 +116,11 @@ func TestNonExistentContainerStatus(t *testing.T) {
 	dt := dockertest.New()
 	c, _ := New(ClientOption(dt.Client()))
 
-	dt.ProvideContainerList(nil, dockertest.NotFoundErr{})
+	dt.ProvideServiceInspectWithRaw(swarm.Service{}, nil, dockertest.NotFoundErr{})
```

Comment:
can you also add lines to check call arguments of _ServiceInspectWithRaw()_ we need to know if _Status()_ calling _ServiceInspectWithRaw()_ with right namespace and options.

Reply:
Done